### PR TITLE
Initial support for use of xcore-win in place of xcore.

### PR DESCRIPTION
**Pull request by augb**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -23,13 +23,13 @@ install:
...
-  - ps: Invoke-WebRequest "$($dl_link)?access_token=$($env:access_token)" -OutFile xcore.zip -Headers @{"Accept"="application/octet-stream"}
-  - 7z e xcore.zip -oxcore
+  # download the latest release of xcore-win so that we can validate the XML files
+  # NOTE: we assume that the first (or only) asset of the release is the xcore-win.zip file
+  - ps: $dl_link = (Invoke-RestMethod "https://api.github.com/repos/openlawlibrary/xcore-win/releases/latest?access_token=$($env:access_token)").assets[0].url
+  - ps: Invoke-WebRequest "$($dl_link)?access_token=$($env:access_token)" -OutFile xcore-win.zip -Headers @{"Accept"="application/octet-stream"}
+  - 7z e xcore-win.zip -oxcore-win
   # run the tests here (instead of AppVeyor's "test" phase, which runs *after* the "build" phase)
-  - xcore\console ..\dc-law-xml
+  - xcore-win\console ..\dc-law-xml ..\dc-law-xml\schemas\dc-library.xsd
```
Comment (joelbarker2011) [inline appveyor.yml]: I'm not sure why the old code references the current project directory (_dc-law-xml_) this way. For better readability/maintainability, let's change to:
`  - xcore-win\console . schemas\dc-library.xsd`

Comment (augb) [inline appveyor.yml]: Will do.

**Comment by joelbarker2011** [comment]
The build is failing because the `access_token` needs to be changed (it's using the `xcore` access token, not the new `xcore-win` access token). This should be the auth token you already have (encrypted via AppVeyor's encrypt tool).

**Comment by joelbarker2011** [comment]
Assigning to @dgreisen. This is ready to merge. The build errors are due to validation issues in the XML.

**Comment by dgreisen** [comment]
The build failed; success!

**Pull request merged**